ui/analyse/css/_context-menu.scss

```diff
@@ -35,7 +35,7 @@
     }
 
     &:hover {
-      background: mix($c-accent, $c-bg-box, 10%);
+      background: $c_accent_bg--mix-10;
```

Comment:
here's the first example of a mix variable. Underscore separates the color variables `accent` and `bg` because a single color variable can also contain hyphens (such as `bg-zebra`. 

Underscore rather than hyphen following the $c to indicate it's a mixed color. We may want to do this the other way around. I mistakenly thought $c_ would be sorted after $c- in browser dev tools but it turns out it's the other way around. So the metric shit ton of mixed colors precedes rather than follows the single color definitions.